prusti-viper/src/encoder/foldunfold/requirements.rs

```diff
@@ -91,13 +91,17 @@ impl RequiredStmtPermissionsGetter for vir::Stmt {
                 ref targets,
                 ..
             }) => {
-                // Preconditions and postconditions are empty
-                assert!(arguments.is_empty());
+                // Preconditions and postconditions are empty or does not require permissions
+                let arguments_perm = arguments
+                    .iter()
+                    .map(|expr| expr.get_required_stmt_permissions(predicates).into_iter())
```

Comment:
What is this for? Does `tick` take arguments? Also this only computes permissions for arguments, not pre-/postconditions. I guess for us that's what we want, because the time resource permissions should not interact with fold/unfold anyway, but the comment should make this clear.

Reply:
This modification was when we had the call to the `tick` method, but as we don't have it anymore we can remove those changes.